pkg/daemon/ceph/osd/volume.go

```diff
@@ -49,6 +50,11 @@ const (
 	dbDeviceFlag         = "--db-devices"
 	cephVolumeCmd        = "ceph-volume"
 	cephVolumeMinDBSize  = 1024 // 1GB
+	restoreConPath       = "/usr/sbin/restorecon"
```

Comment:
If c-v thinks this command should be run, how can we just skip it with no consequences? The containers aren't affected?

Reply:
Right, c-v just runs the command and pursues if the exit code is 0. So this has no consequence.